var/spack/repos/builtin/packages/brigand/package.py

```diff
@@ -23,6 +23,7 @@ class Brigand(CMakePackage):
     version("1.1.0", sha256="afdcc6909ebff6994269d3039c31698c2b511a70280072f73382b26855221f64")
     version("1.0.0", sha256="8daf7686ff39792f851ef1977323808b80aab31c5f38ef0ba4e6a8faae491f8d")
 
+    depends_on("c", type="build")  # generated
```

Comment:
There is no C in brigand, it is a full c++ header only library so this modification sounds incorrect.

Reply:
https://github.com/edouarda/brigand/blob/master/CMakeLists.txt#L3

https://cmake.org/cmake/help/latest/command/project.html

"By default C and CXX are enabled if no language options are given. Specify language NONE, or use the LANGUAGES keyword and list no languages, to skip enabling any languages."

Reply:
Ok so that is on us to fix our cmake.